### DISCUSSION TITLE
Pinch Gesture: initial offset scale value always starts with 1

### DISCUSSION THREAD

**Discussion by zaki-hanafiah** [pmndrs/use-gesture#589] [Q&A] [answered]
**Describe the bug**
Initial scale offset value from the usePinch gesture state always starts from `1.0`. The issue I'm facing for my use-case is the element has an initial scale differing from `1.0`, and it will result in a sort of sudden zoom transition on the first user pinch interaction. 
Is there an existing way to define the initial scale value so that it reflects in the returned offset scale? 
In the the example sandbox link below, the initial scale is set in the `useSpring` style hook.

**Sandbox or Video**
Video: 
![Video](https://user-images.githubusercontent.com/31648832/227113927-4df1418f-9eec-4abf-9181-eed970a5efa3.mp4)

Sandbox link (Derived from the pinch / zoom demo):
https://codesandbox.io/s/withered-paper-j10mv4?file=/src/App.tsx 

**Information:**
 - Use Gesture version: 10.2.24
 - Device: iPhone 14
 - OS: iOS 16.2
 - Browser: Safari

**Checklist:**
  - [x ] I've read the [documentation](https://use-gesture.netlify.app/).
  - [ ] If this is an issue with drag, I've tried setting `touch-action: none` to the draggable element.

**Comment by dbismut** [comment]
https://use-gesture.netlify.app/docs/options/#from

**Comment by zaki-hanafiah** [comment, marked as answer]
In case anyone else runs into the same problem:

I used `from` in the `usePinch` config via: `from: () => [scaleValue, rotationValue]` to set the initial scale value.

**Reply by dbismut** [reply]
The second element of the array is rotation. You shouldn't use scale as its initial value.

**Reply by zaki-hanafiah** [reply]
Thanks, have edited.
